html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-optional-001.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- position features `top` and `left`</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+
+<!-- user agents are not required to support open features other than `noopener`
+     and on some platforms position and size features don't make sense -->
+<meta name="flags" content="may" />
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/PrefixedPostMessage.js"></script>
+<script>
+var windowURL = 'resources/message-opener.html';
+
+[ 'left=152',
+  ' left = 152',
+  'left==152',
+  '\nleft= 152',
+  ',left=152,,',
+  'LEFT=152'
+].forEach((features, idx, arr) => {
+  async_test(t => {
+    var prefixedMessage = new PrefixedMessageTest();
+    prefixedMessage.onMessage(t.step_func_done((data, e) => {
+      e.source.close();
+      assert_equals(data.left, 152);
+    }));
+    var win = window.open(prefixedMessage.url(windowURL), '', features);
+  }, `"${features}" (${idx + 1} of ${arr.length}) should set left position of opened window`);
```

Comment:
Remove index/length from the test name. Removing or adding tests shouldn't cause other tests to be renamed.